src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/AnalysisDefaults.scala

```diff
@@ -19,6 +19,7 @@ private[analyzer] object AnalysisDefaults {
     with EnableLocation[T]
     with EnableEntity[T]
     with EnableLanguage[T]
+    with FilterBlacklist[T]
```

Comment:
What about `EnableBlacklist` or something that starts with `Enable` for consistency?